frontend/src/components/prompt/PopoverPrompt.vue

```diff
@@ -12,7 +12,7 @@
     <template #activator="{ attrs, on }">
       <slot name="activator" v-bind="{ attrs, on }" />
     </template>
-    <div class="ec-activator" @click="open = false">
+    <div class="ec-activator v-card__actions pa-0" @click="open = false">
```

Comment:
I can't tell the difference

Reply:
Before:
<img width="628" alt="Bildschirmfoto 2024-01-04 um 08 56 30" src="https://github.com/ecamp/ecamp3/assets/3001985/6e2d6021-b6f2-425c-a384-b52b17578d74">


After:
<img width="639" alt="Bildschirmfoto 2024-01-04 um 08 55 42" src="https://github.com/ecamp/ecamp3/assets/3001985/212a3f1f-b24a-4894-b4f6-458f3d0adf5c">

The button should now be the same width as the activator button on desktop